cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -175,7 +175,7 @@ func (i *deployCommand) InitHook() {
 	i.command.Flags().StringVar(&i.flags.projectGroupId, "project-group-id", "", "Indicates the unique identifier of the organization or group that created the project, e.g: 'com.company'")
 	i.command.Flags().StringVar(&i.flags.projectArtifactId, "project-artifact-id", "", "Indicates the unique base name of the primary artifact being generated, e.g: 'project'")
 	i.command.Flags().StringVar(&i.flags.projectVersion, "project-version", "", "Indicates the version of the artifact generated by the project, e.g: '1.0-SNAPSHOT'")
-  i.command.Flags().BoolVar(&i.flags.enableMavenDownloadOutput, "enable-maven-download-output", false, "If set to true will print the logs for downloading/uploading of maven dependencies. Defaults to false")
+	i.command.Flags().BoolVar(&i.flags.enableMavenDownloadOutput, "enable-maven-download-output", false, "If set to true will print the logs for downloading/uploading of maven dependencies. Defaults to false")
```

Comment:
why not only maven-output?
and, maybe use BoolVarP to set a short-hand option for this as well?

Reply:
I had changes name to **maven-output**. I didn't find any single character which we can use here as short hand. I check "o" but its already taken. Could you please suggest which we short hand character we can use here?

Reply:
using "m" as short hand char